ui/src/app/components/apps-panel/expanded-app.spec.tsx

```diff
@@ -210,10 +210,26 @@ describe('ExpandedApp', () => {
 
   const gkeAppTypes = [UIAppType.CROMWELL, UIAppType.RSTUDIO, UIAppType.SAS];
   describe.each(gkeAppTypes)('GKE App %s', (appType) => {
-    test.each([
-      [AppStatus.RUNNING, 'Pause'],
-      [AppStatus.STOPPED, 'Resume'],
+    test.each([[AppStatus.RUNNING, 'Pause']])(
```

Comment:
why remove the Resume cases from these?

Reply:
That was a mistake. Thanks for mentioning it.